src/components/RightPane/SectionTable/SectionTableButtons.tsx

```diff
@@ -109,6 +128,9 @@ export const ScheduleAddCell = withStyles(styles)((props: ScheduleAddCellProps)
                     <MenuItem onClick={() => closeAndAddCourse(scheduleNames.length, true)}>
                         Add to All Schedules
                     </MenuItem>
+                    <MenuItem onClick={() => closeCopyAndAlert()}>
```

Comment:
I think this should suffice. In general I like to create handler functions that can be directly called by the event so I don't have to inline a function definition.
```suggestion
                    <MenuItem onClick={closeCopyAndAlert}>
```